pkg/cli/cliflags/flags.go

```diff
@@ -324,6 +324,15 @@ disabled by setting the environment variable COCKROACH_SKIP_KEY_PERMISSION_CHECK
 		Description: `Path to the CA key.`,
 	}
 
+	MaxOffset = FlagInfo{
+		Name: "max-offset",
+		Description: `
+Maximum allowed clock offset for the cluster. If observed clock offsets exceed
```

Comment:
Done.

And regarding our offline chat about printing the default value, that already gets printed by the CLI machinery by default:

```
$ ./cockroach start --help
[...]
      --max-offset duration
        Maximum allowed clock offset for the cluster. If observed clock offsets
        exceed this limit, servers will crash to minimize the likelihood of
        reading inconsistent data. Increasing this value will increase the time
        to recovery of failures as well as the frequency of uncertainty-based
        read restarts.

        Note that this value must be the same on all nodes in the cluster.
        In order to change it, every node in the cluster must be stopped and
        restarted with the new value.
        (default 500ms)
[...]
`